proto/finschia/zkauth/v1beta1/query.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+package finschia.zkauth.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+
+import "finschia/zkauth/v1beta1/params.proto";
+
+option go_package = "github.com/Finschia/finschia-sdk/x/zkauth/types";
+
+// Query defines the gRPC querier service.
+service Query {
+    // Parameters queries the parameters of the module.
+    rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
+        option (google.api.http).get = "/finschia/zkauth/v1beta1/params";
+  }
+
+}
+
+// QueryParamsRequest is request type for the Query/Params RPC method.
+message QueryParamsRequest {}
+
+// QueryParamsResponse is response type for the Query/Params RPC method.
+message QueryParamsResponse {
+    // params holds all the parameters of this module.
+    Params params = 1 [(gogoproto.nullable) = false];
+}
```

Comment:
If there is not `params` of genesis, this query does not need.

Reply:
Yes, you are right.